src/sidebar/store/create-store.js

```diff
@@ -241,7 +260,7 @@ export function createStore(modules, initArgs = [], middleware = []) {
   }
   Object.assign(store, selectorMethods);
 
-  return store;
+  return /** @type {any} */ (store);
```

Comment:
Can you help me understand this `any`?

Reply:
This is basically saying "trust me" to TS, since it can't figure out for itself that the object we've put together has the shape we promise.